airbyte-integrations/connectors/source-chartmogul/source_chartmogul/source.py

```diff
@@ -70,6 +72,55 @@ def path(self, **kwargs) -> str:
         return "v1/activities"
 
 
+class CustomerCount(ChartmogulStream):
+    primary_key = "date"
+
+    # customer-count API requires start date.
+    default_start_date = "2020-01-01"
```

Comment:
Would be possible to request from users to set the start date OR get from the API the minimal start date from the api token used?

Reply:
The Spec request a yyyy-mm-ddT00:00:00Z format and here you're using a yyyy-mm-dd, this could cause any trouble when doing the requests?

Reply:
I decided to make _start_date_ required. It should be easier to manage going forward. API accept both date or timestamp.